packages/fluentui/react-builder/src/components/ComponentPropertiesPanel.tsx

```diff
@@ -0,0 +1,63 @@
+import * as React from 'react';
+import { AccessibilityError } from '../accessibility/types';
+import { ComponentInfo } from '../componentInfo/types';
+import { Description } from './Description';
+import { Knobs } from './Knobs';
+import { DesignerMode, JSONTreeElement } from './types';
+
+export type ComponentPropertiesPanelProps = {
+  accessibilityErrors: AccessibilityError[];
+  mode: DesignerMode;
+  onPropUpdate: ({ jsonTreeElement: JSONTreeElement }) => void;
+  onPropChange: ({
+    jsonTreeElement,
```

Comment:
This is the same type as in `Builder.tsx` and can be re-used here. See comment in `Builder.tsx`.